frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -86,17 +137,6 @@ export const defaultYAMLPodNetworkingInterface = {
   network: 'Pod Networking',
 };
 
-// Fake windows machine, still cirros in the heart
-export const windowsVMConfig: BaseVMConfig = {
-  operatingSystem: OperatingSystem.WINDOWS_10,
-  flavorConfig: { flavor: Flavor.MEDIUM },
-  workloadProfile: WorkloadProfile.DESKTOP,
-  sourceURL:
-    'http://cnv-qe-server.rhevdev.lab.eng.rdu2.redhat.com/files/files-https/cirros/cirros-qcow2.img',
-  sourceContainer: 'kubevirt/cirros-registry-disk-demo',
-  cloudInitScript: `#cloud-config\nuser: cloud-user\npassword: atomic\nchpasswd: {expire: False}\nhostname: vm-${testName}`, // reusing cirros
-};
-
 export const multusNetworkInterface = {
```

Comment:
not sure if not freezing these is a big issue?

Reply:
Will freeze